buildkite/bazelci.py

```diff
@@ -1819,9 +1884,26 @@ def main(argv=None):
             )
         elif args.subparsers_name == "runner":
             configs = fetch_configs(args.http_config, args.file_config)
+            tasks = configs.get("tasks", {})
+            task_config = tasks.get(args.task)
+            if not task_config:
+                raise BuildkiteException(
+                    "No such task '{}' in configuration. Available: {}".format(
+                        args.task, ", ".join(tasks)
+                    )
+                )
+
+            platform = get_platform_for_task(args.task, task_config)
+            if not platform:
```

Comment:
This is never the case due to the fallback to the task id in case the `platform` field is missing.